### ISSUE TITLE
Build fails with Rebar3

### ISSUE THREAD

**Issue by AndriiKlymchuk** [processone/eimp#7]
Hi! When building on Ubuntu 16.04 with Rebar3 I get next error:
```
===> Fetching eimp ({git,"https://github.com/processone/eimp.git",
                                {branch,"master"}})
===> Uncaught error in rebar_core. Run with DEBUG=1 to see stacktrace or consult rebar3.crashdump
===> When submitting a bug report, please include the output of `rebar3 report "your command"`
Makefile:6: recipe for target 'compile' failed
make: *** [compile] Error 1
```

In ebar3.crashdump I have

```
Error: {badmatch,
           {error,
               {228,file,
                {error,
                    {badmatch,["3","5","0+build","3974","refb5d7d35"]},
                    [{erl_eval,expr,3,[]}]}}}}
[{rebar_config,consult_file_,1,
               [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_config.erl"},
                {line,230}]},
 {rebar_config,consult_file,1,
               [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_config.erl"},
                {line,212}]},
 {rebar_prv_install_deps,handle_dep,6,
                         [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_prv_install_deps.erl"},
                          {line,280}]},
 {rebar_prv_install_deps,update_unseen_dep,9,
                         [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_prv_install_deps.erl"},
                          {line,273}]},
 {lists,foldl,3,[{file,"lists.erl"},{line,1263}]},
 {rebar_prv_install_deps,handle_profile_level,7,
                         [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_prv_install_deps.erl"},
                          {line,183}]},
 {rebar_prv_install_deps,do_,1,
                         [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_prv_install_deps.erl"},
                          {line,82}]},
 {rebar_core,do,2,
             [{file,"/home/andrew/rebar3_test/_build/default/lib/rebar/src/rebar_core.erl"},
              {line,154}]}]
```

Rebar3 and Erlang versions

```
rebar 3.5.0+build.3974.refb5d7d35 on Erlang/OTP 20 Erts 9.1.5
```
How can I fix it? Thanks in advance.

**Comment by prefiks** [comment]
This should be fixed by ff7dfd1896aa41ea69e60ef0966d099b4fbfe505

**Comment by AndriiKlymchuk** [comment]
Wow, you're super fast, thanks!
